pytest_cli.py

```diff
@@ -0,0 +1,41 @@
+from collections.abc import Iterable
```

Comment:
I would rather add `#!/usr/bin/env python3` as the 1st line here and make this file executable.